doc/manual/src/glossary.md

```diff
@@ -168,9 +181,10 @@
 
   A high-level description of software packages and compositions
   thereof. Deploying software using Nix entails writing Nix
-  expressions for your packages. Nix expressions are translated to
-  derivations that are stored in the Nix store. These derivations can
-  then be built.
+  expressions for your packages. Nix expressions specify [derivations][derivation],
+  which are [instantiated][instantiate] into the Nix store as [store derivations][store derivation].
+  These derivations can then be [realised][realise] (built) to produce
```

Comment:
```suggestion
  These derivations can then be [realised][realise] to produce
```
Maybe I'm getting paranoid, but I've spent so much effort to pry a proper definition for realisation out of the code and everyone else's minds, that I'm careful not to throw around words that aren't well defined. To "build" is a colloquialism for "running the `builder` executable of a derivation`, and that's just one possible way of realising it.